topologies/100_000_spans_with_20_fixed_tags_per_second.json

```diff
@@ -0,0 +1,199 @@
+          "routes" : [
+            {
+              "route" : "/7",
+              "downstreamCalls" : { }
+            }
+          ]
+        },
+        {
+          "serviceName" : "spanFiller8",
+          "instances" : [ "spanfiller-6b654dbf57-00000" ],
+          "tagSets": [{"tags" : { "fix-tag-name-00" : "fix-tag-value-00", "fix-tag-name-01" : "fix-tag-value-01", "fix-tag-name-02" : "fix-tag-value-02", "fix-tag-name-03" : "fix-tag-value-03", "fix-tag-name-04" : "fix-tag-value-04", "fix-tag-name-05" : "fix-tag-value-05", "fix-tag-name-06" : "fix-tag-value-06", "fix-tag-name-07" : "fix-tag-value-07", "fix-tag-name-08" : "fix-tag-value-08", "fix-tag-name-09" : "fix-tag-value-09", "fix-tag-name-10" : "fix-tag-value-10", "fix-tag-name-11" : "fix-tag-value-11", "fix-tag-name-12" : "fix-tag-value-12", "fix-tag-name-13" : "fix-tag-value-13", "fix-tag-name-14" : "fix-tag-value-14", "fix-tag-name-15" : "fix-tag-value-15", "fix-tag-name-16" : "fix-tag-value-16", "fix-tag-name-17" : "fix-tag-value-17", "fix-tag-name-18" : "fix-tag-value-18", "fix-tag-name-19" : "fix-tag-value-19" }, "maxLatency":  1}],
+          "routes" : [
+            {
+              "route" : "/8",
+              "downstreamCalls" : { }
+            }
+          ]
+        },
+        {
+          "serviceName" : "spanFiller9",
+          "instances" : [ "spanfiller-6b654dbf57-00000" ],
+          "tagSets": [{"tags" : { "fix-tag-name-00" : "fix-tag-value-00", "fix-tag-name-01" : "fix-tag-value-01", "fix-tag-name-02" : "fix-tag-value-02", "fix-tag-name-03" : "fix-tag-value-03", "fix-tag-name-04" : "fix-tag-value-04", "fix-tag-name-05" : "fix-tag-value-05", "fix-tag-name-06" : "fix-tag-value-06", "fix-tag-name-07" : "fix-tag-value-07", "fix-tag-name-08" : "fix-tag-value-08", "fix-tag-name-09" : "fix-tag-value-09", "fix-tag-name-10" : "fix-tag-value-10", "fix-tag-name-11" : "fix-tag-value-11", "fix-tag-name-12" : "fix-tag-value-12", "fix-tag-name-13" : "fix-tag-value-13", "fix-tag-name-14" : "fix-tag-value-14", "fix-tag-name-15" : "fix-tag-value-15", "fix-tag-name-16" : "fix-tag-value-16", "fix-tag-name-17" : "fix-tag-value-17", "fix-tag-name-18" : "fix-tag-value-18", "fix-tag-name-19" : "fix-tag-value-19" }, "maxLatency":  1}],
```

Comment:
use a tag generator instead 
```
"tagGenerators": [{"numTags": 32, "numVals": 152, "valLength": 64}]
```

Reply:
This is what I have in https://github.com/Omnition/synthetic-load-generator/blob/master/topologies/100_000_spans_with_20_tags_per_second.json, however, I noticed that the rate that it generates doesn't match my expectation and if I use fixed I get a higher rate of spans/sec. Here is what I have in the other file: 
```json
"tagSets": [ { "weight": 1, "maxLatency": 350, "tagGenerators": [{"numTags": 20, "numVals":  100, "valLength": 64}]} ],
```

Reply:
Ah I see, yea using the tag generator is more expensive (since it does work to generate the tags).